include/api.php

```diff
@@ -3574,24 +3575,24 @@ function api_friendica_group_delete($type) {
 
 		// error if no gid specified
 		if ($gid == 0 || $name == "")
-			throw new BadRequestException('gid or name not specified');
+			throw new HTTP\BadRequestException('gid or name not specified');
 
 		// get data of the specified group id
 		$r = q("SELECT * FROM `group` WHERE `uid` = %d AND `id` = %d",
 			intval($uid),
 			intval($gid));
 		// error message if specified gid is not in database
 		if (!dbm::is_result($r))
-			throw new BadRequestException('gid not available');
+			throw new HTTP\BadRequestException('gid not available');
 
 		// get data of the specified group id and group name
 		$rname = q("SELECT * FROM `group` WHERE `uid` = %d AND `id` = %d AND `name` = '%s'",
 			intval($uid),
 			intval($gid),
 			dbesc($name));
 		// error message if specified gid is not in database
-		if (!dbm::is_result($rname))
-			throw new BadRequestException('wrong group name');
+		if (count($rname) == 0)
```

Comment:
Standards: Please add braces to this conditional statement and keep the `dbm::is_result()` use.